classes/Http/Controller/ProfileController.php

```diff
@@ -40,16 +41,33 @@ class ProfileController extends BaseController
      */
     private $profileImageProcessor;
 
+    /**
+     * @var PathInterface
+     */
+    private $path;
+
+    /**
+     * ProfileController constructor.
+     *
+     * @param Authentication        $authentication
+     * @param HTMLPurifier          $purifier
+     * @param ProfileImageProcessor $profileImageProcessor
+     * @param Twig_Environment      $twig
+     * @param UrlGeneratorInterface $urlGenerator
+     * @param PathInterface         $path
+     */
```

Comment:
As this docblock only repeats the arguments and their types, which - as far as I can tell - are already sufficiently declared, can we remove the docblock?